app/main.py

```diff
@@ -1,20 +1,39 @@
-from typing import Optional
-from fastapi import Body, FastAPI
-from pydantic import BaseModel
+from fastapi import Depends, FastAPI, status, HTTPException
+from sqlalchemy.orm import Session
+from sqlalchemy import text
+
+from app.models import User
+from .database.commands import db_create_user, db_get_users, db_get_user
+from .database import db_models
+from .database.setup import engine, get_db
 
 fastapi = FastAPI()
 
-class User(BaseModel):
-    username: str
-    password: str
-    email: Optional[str] = None
-    phone_number: Optional[int] = None
+get_db()
+db_models.Base.metadata.create_all(bind=engine)
```

Comment:
Depending on how this is meant to be run/deployed you can either make this called `__main__py` and move the function definitions elsewhere or add this code under:
```python
if __name__ == "__main__":
    do_my_stuff()
```